src/core/GOBitmap.h

```diff
@@ -11,30 +12,31 @@
 
 class wxImage;
 
-class GOBitmap
-{
-private:
-	wxImage* m_img;
-	wxBitmap m_bmp;
-	double m_Scale;
-	int m_ResultWidth;
-	int m_ResultHeight;
-	unsigned m_ResultXOffset;
-	unsigned m_ResultYOffset;
+class GOBitmap {
+ private:
+  wxImage* m_img;
+  wxBitmap m_bmp;
+  double m_Scale;
+  int m_ResultWidth;
+  int m_ResultHeight;
+  unsigned m_ResultXOffset;
+  unsigned m_ResultYOffset;
 
-	void ScaleBMP(wxImage& img, double scale, const wxRect& rect, GOBitmap* background);
+  void ScaleBMP(wxImage& img, double scale, const wxRect& rect,
+                GOBitmap* background);
```

Comment:
Here I receive
```
  void ScaleBMP(
    wxImage &img, double scale, const wxRect &rect, GOBitmap *background);
```

Seems the options 

```
AlignAfterOpenBracket: AlwaysBreak
AlignOperands: 'false'
BinPackArguments: 'false'
BinPackParameters: 'false'
```

aren't not work with your clang. Or your clang-format ignores the `.clang-format` file

Reply:
Oh yes I see.

Ok, I will try to force the config file

Reply:
@oleg68 Ok, I got it. The style attribute should point file to take that one into account.

Reply:
Is this ok for you ?

But reading the clang-documentation, it recommends to do a clang-format dump-config to generate a complete setting file (A lot of config entries instead of this for the moment).

I can try it.

Reply:
> I can try it.

Done in last commit. It seems good. 
I've changed the repo for clang as the last version of clang-format is on pre-commit one.